docs/style-guide.rst

```diff
@@ -112,6 +112,157 @@ No::
         }
     }
 
+Maximum Line Length
+===================
+
+Keeping lines under the `PEP 8 recommendation <https://www.python.org/dev/peps/pep-0008/#maximum-line-length>`_ of 79 (or 99) 
+characters helps readers easily parse the code.
+
+Wrapped lines should conform to the following guidelines.
+
+1. The first argument should not be attached to the opening parenthesis. 
+
+2. One, and only one, indent should be used.
+
+3. Each argument should fall on its own line.
+
+4. The terminating element, :code:`)` or :code:`);`, should be placed on the final line by itself.
+
+Function Calls
+
+Yes::
+
+    thisFunctionCallIsReallyLong.(
```

Comment:
What are the dots doing after each function name?

Reply:
nothing. will delete.